python/mpt/tests/src/mpt/test_utils.py

```diff
@@ -0,0 +1,23 @@
+from ethereum.cancun.trie import BranchNode, bytes_to_nibble_list
+from ethereum_types.bytes import Bytes
+from hypothesis import given
+
+from cairo_addons.testing.errors import strict_raises
+from mpt.utils import check_branch_node, nibble_list_to_bytes
+
+
+class TestUtils:
+    @given(bytes=...)
+    def test_nibble_list_to_bytes(self, bytes: Bytes):
+        assert nibble_list_to_bytes(bytes_to_nibble_list(bytes)) == bytes
+
+    @given(branch_node=...)
+    def test_check_branch_node(self, cairo_run, branch_node: BranchNode):
```

Comment:
are you generating invalid branches with the strategy?

Reply:
not yet